### ISSUE TITLE
Weird sub-album renaming behavior

### ISSUE THREAD

**Issue by rafaelkarrer** [LycheeOrg/Lychee-Laravel#175]
- Right-click a sub-album and select rename.
- The dialog window will show the parent album's name (not correct).
- Change the string to the desired name and hit "Set Title" --> At first, the name of the sub-album does not change, however, the parent album name in the title bar incorrectly gets updated to the previously entered sub-album name.
- Re-load the page or step out and into the album again --> All names are displayed correctly now. Looks like there is something happening with the display of album names during and after the rename.

Renaming from inside the sub-album works correctly though.

Tested using d2819d4a0193d87498c23dc82cb77838127155d8

**Comment by kamil4** [comment]
Yes, there are a number of such problems in the front end at the moment. Subalbums, as well as multi-user support, are major new features in v4, and while they are functional at this point, there are a number of corner cases that need improvements.

I've been compiling a list of issues with these new features and while I wasn't aware of the particular one you reported, I did encounter its "close relative" when one tries to delete a subalbum via right-click. If you encounter any others, do report them by opening a new issue for each one. We will get to them eventually.
